scrapers/BlackFoxBound.yml

```diff
@@ -0,0 +1,31 @@
+name: Blackfoxbound UK
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - blackfoxbound.net
+    scraper: sceneScraper
+
+xPathScrapers:
+  sceneScraper:
+    scene:
+      Title: //meta[@property="og:title"]/@content
+      Date:
+        selector: //div[@class="page-header"]//span[@title="Published at"]//text()
+        postProcess:
+          - parseDate: 2006-01-02
+      Details:
+        selector: //meta[@property="og:description"]/@content
+      Performers:
+        Name:
+          selector: //span[@title="Models"]//a/text()
+      Tags:
+        Name:
+          selector: //span[@title="Categories"]//a/text()
+      #Image: //div[@class="video-container"]//video/@poster
+      Image: //meta[@property="og:image"]/@content
```

Comment:
You can have both of these if you OR them together, in case some scenes have video posters and some have opengraph image tags
`//div[@class="video-container"]//video/@poster|//meta[@property="og:image"]/@content`

Poking around at a few of these scenes it looks like `//div[contains(@id, "video")]/@poster` would also work and it seems to grab higher resolution images (unfortunately in many cases they're just upscaled)